site/content/docs/5.2/utilities/text.md

```diff
@@ -38,7 +38,7 @@ Wrap text with a `.text-wrap` class.
 Prevent text from wrapping with a `.text-nowrap` class.
 
 {{< example >}}
-<div class="text-nowrap bg-light border" style="width: 8rem;">
+<div class="text-nowrap bg-secondary border" style="width: 8rem;">
```

Comment:
```suggestion
<div class="text-nowrap bg-body-secondary border" style="width: 8rem;">
```

I'd rather use something like `.bg-body-secondary` to avoid a11y color contrast issue; especially in light mode with `.bg-secondary`:

![Screenshot 2022-11-17 at 08 42 08](https://user-images.githubusercontent.com/17381666/202386011-f06d50fa-b8d6-4ed8-b263-53cd236ec4f3.png)